docs/labs/index.rst

```diff
@@ -15,17 +15,6 @@ If we get lots of positive feedback about any one of these projects then we will
 
 .. _labs-demo:
 
-Flocker Docker plugin with CLI and GUI
```

Comment:
The only change that needs to happen to this section is that `:ref:\`Flocker Docker plugin <labs-docker-plugin>\`` needs to be replaced with `:ref:\`Flocker Docker plugin <docker-plugin>\``.